vars/transportRequestUploadFile.groovy

```diff
@@ -14,10 +14,10 @@ import hudson.AbortException
 
 @Field Set generalConfigurationKeys = [
     'changeManagement',
+    'applicationId'
```

Comment:
not aligned with PR title, remove here and move to `parameterKeys` and `stepConfigurationKeys`